geniza/corpus/templates/corpus/document_list.html

```diff
@@ -0,0 +1,59 @@
+{% extends 'base.html' %}
+{% load static i18n %}
+
+{% comment %}Translators: Title for document search page{% endcomment %}
+{% translate 'Search Documents' as doctitle%}
+
+{% block title %}{{ doctitle }}{% endblock title %}
+
+{% block main %}
+<div class="container search">
+    <h1>{{ doctitle }}</h1>
+
+    <form>
+    {{ form }}
+    {% for field, errors in form.errors.items %}
+    {# no clean way to get field labels here, so we just collapse the error lists #}
+        {% for error in errors %}
+            <li>{{ error }}</li>
+        {% endfor %}
+    {% endfor %}
+    </form>
```

Comment:
could we get a "go" button or similar? might end up hiding it later depending on the implementation, but it's nice to have for now (not everyone will think to press <kbd>enter</kbd>) and we'll need it later regardless